int8[] costmap_ros
int32 width
int32 height
geometry_msgs/Point[] start
geometry_msgs/Point[] goal 
---
PathArray[] plan_array
